script/main_test.py

```diff
@@ -41,6 +41,32 @@ def tearDown(self):
     self.connection.close()
 
 
+class TestUsersBorn(unittest.TestCase):
+
+  def setUp(self):
+    self.connection = create_connection("../db/pythonsqliteusers.db")
+
+  def test_users_born(self):
+    self.assertEqual(len(main.users_born(self.connection, '1944-01-01', '1999-01-01')), 1000)
+    self.assertEqual(len(main.users_born(self.connection, '1960-01-01', '1965-01-01')), 92)
+    self.assertEqual(len(main.users_born(self.connection, '1990-05-11', '1991-06-10')), 21)
+
+  def tearDown(self):
+    self.connection.close()
+
+
+class TestMostSecurePasswords(unittest.TestCase):
```

Comment:
You don't need to create separate class for each test. Use one class (think about good name for this class). And then use separate functions.